apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -80,11 +80,20 @@ router.get(
 
       const stringifier = stringifyStream({
         header: true,
-        columns: ['Time', 'Auth user', 'Event', 'Instructor question', 'Student question', 'Data'],
+        columns: [
+          'Time',
+          'Auth user',
+          'Fingerprint ID',
```

Comment:
Nit: let's just call this "Fingerprint" for now, to match the UI and because this isn't actually an ID.

```suggestion
          'Fingerprint',
```